src/itemviewformaction.cpp

```diff
@@ -117,6 +118,19 @@ void ItemViewFormAction::prepare()
 					rxman,
 					"article");
 		} else {
+			// Add enclosure url to link list.
+			if (item->enclosure_type() != "") {
```

Comment:
Nitpick: it's better to use `empty()` method of `std::string`, because it works in constant time. `x != ""`, on the other hand, constructs `std::string` from literal `""` and then does char-by-char comparison, which involves dereferencing two pointers (one in each `std::string`).